modules/orchestrator/src/main/java/org/eclipse/xpanse/orchestrator/credential/CredentialCenter.java

```diff
@@ -88,4 +112,31 @@ AbstractCredentialInfo getCredentialByType(Csp csp, String userName,
                                                CredentialType credentialType) {
         return null;
     }
+
+    /**
+     * Get credentialInfo from the environment using Csp.
+     *
+     * @param csp The csp to get.
+     */
+    private AbstractCredentialInfo findCredentialInfoFromEnv(Csp csp) {
+        List<AbstractCredentialInfo> credentialAbilities = getCredentialAbilities(csp);
+        for (AbstractCredentialInfo credentialAbility : credentialAbilities) {
+            CredentialDefinition credentialDefinition = (CredentialDefinition) credentialAbility;
+            List<CredentialVariable> variables = credentialDefinition.getVariables();
+            boolean availableCredentialFound = true;
+            for (CredentialVariable variable : variables) {
+                String envValue = System.getenv(variable.getName());
+                if (envValue == null) {
```

Comment:
Here is no need to use this variable 'availableCredentialFound'.  You can directly set the environment variable value when that is not blank. if (StringUtils.isNotBlank(envValue)){variable.setValue(envValue);}

Reply:
because we only need to obtain one type of credential information, when availableCredentialFound is true, a loop will break out and we will no longer search for it from the environment, based on your suggestion, I have made optimization here.

Reply:
Ok.